packages/api/src/hooks/useTransferAccounts.ts

```diff
@@ -0,0 +1,16 @@
+import useQuery from '../useQuery';
+
+/** A custom hook that gets the list of all transferable accounts of the user. */
+const useTransferAccounts = () => {
```

Comment:
Cant we use `useTransferBetweenAccounts` hook? its a mutation hook so you can pass payload

Reply:
sorted